tests/common.py

```diff
@@ -41,6 +42,12 @@ def get_110_ccc_ff():
     return forcefield
 
 
+def fixed_overflowed(a):
+    """Refer to timemachine/cpp/src/kernels/k_fixed_point.hpp::FLOAT_TO_FIXED_ENERGY for documentation onOne how we handle energies and overflows"""
+    converted_a = np.int64(np.uint64(a))
```

Comment:
We should make this function a bit more precise.

Doesn't this always generate a number in the closed interval `[np.iinfo(np.int64).min, np.iinfo(np.int64).max]`?  I.e. I don't think it's possible to have `<=`, or `>=` simply testing for equality should suffice:

`converted_a == np.iinfo(np.int64).min or converted_a == np.iinfo(np.int64).max`

Alternatively, if `u_fixed` is a `int128`, then there shouldn't be a need to cast

`a <= np.iinfo(np.int64).min or a >= np.iinfo(np.int64).max`


Another nit:

When do we actually trigger the the `min` case, I thought we always converted to `MAX_INT64` in the case of `nans/-inf/+inf` etc. Should we just throw an assertion if we did hit `MIN_INT64`?

Reply:
Changed to check equality here 080e54a60c1eefc46080c7c1b0cfa8bd1ea8c2e2

Still need to add assertion

Reply:
Added assertion for MIN_INT64 in https://github.com/proteneer/timemachine/pull/1090/commits/0665bf77d57027ae164519872759648835dbbc14